Split/Storage/split_cache.xcdatamodeld/split_cache.xcdatamodel/contents

```diff
@@ -12,9 +12,9 @@
         </uniquenessConstraints>
     </entity>
     <entity name="GeneralInfo" representedClassName="GeneralInfoEntity" syncable="YES">
-        <attribute name="longValue" attributeType="Integer 64" defaultValueString="0" usesScalarValueType="YES"/>
+        <attribute name="longValue" optional="YES" attributeType="Integer 64" defaultValueString="0" usesScalarValueType="YES"/>
```

Comment:
Javi, take a look to this because according to `GeneralInfoEntity` class, longValue is mandatory and stringValue is optional.